src/pages/index.js

```diff
@@ -83,20 +71,65 @@ const editProfileFormValidator = new FormValidator(
   settings,
   editProfileModalFormElement
 );
+const editAvatarFormValidator = new FormValidator(
+  settings,
+  editAvatarModalFormElement
+);
+const api = new Api({
+  baseUrl: "https://around.nomoreparties.co/v1/cohort-3-en",
+  headers: {
+    authorization: "b685d3e0-616a-4dae-bc5b-53892a4f7953",
+    "Content-Type": "application/json",
+  },
+});
+function createCard(item) {
+  // create instance of Card class
+  const card = new Card(
+    item,
+    "#card-template",
+    onCardClick,
+    handleCardDeleteFunctionInIndexComponent
+  );
+  //create a card by calling getCardElement method from Card class
+  const cardElement = card.getCardElement();
+  //return the card
+  return cardElement;
+}
+
+api.getUserInfo().then((data) => {
+  const name = data.name;
+  const subtitle = data.about;
+  const url = data.avatar;
+  userInfo.setUserInfo({ name, subtitle });
+  userInfo.setNewAvatar(url);
+});
+
+api
+  .getInitialCards()
```

Comment:
You need to combine the requests of the user's info and the cards together in `Promise.all` because it might happen so that the `_id` has not been received yet but the cards have and your cards will be rendered not correctly. 
More info is here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all

```js

Promise.all([getUserInfo(), getCards()])
// destructure the response
  .then(([userData, cards]) => {
      // set all the data
  })
  .catch(err => {
    // catch possible errors
  });

```